pydrive2/__pyinstaller/test_hook-googleapiclient.py

```diff
@@ -3,12 +3,17 @@
 from PyInstaller import __main__ as pyi_main
 
 
-_APP_SOURCE = """import importlib.resources
+_APP_SOURCE = """
+import sys
+if sys.version_info >= (3, 10):
+    from importlib.resources import contents
+else:
+    from importlib_resources import contents
```

Comment:
`importlib.resources.contents` is already available in 3.7+, but due to the way pyinstaller handles importlib, we have to use the `importlib_resources` backport versions on <3.10 for this test

Reply:
@pmrowla Would it be worth putting a comment there?

Reply:
I'll wait and see if this CI run passes and then add the comment before merging